mmhuman3d/__init__.py

```diff
@@ -16,7 +16,7 @@ def digit_version(version_str):
 
 
 mmcv_minimum_version = '1.3.17'
-mmcv_maximum_version = '1.6.0'
+mmcv_maximum_version = '1.6.1'
```

Comment:
Would this affect others?

Reply:
We have to update mmcv_maximum_version otherwise the unit tests will fail. As for whether it affects others, I don't think it will.